distributions/otelcol-contrib/manifest.yaml

```diff
@@ -0,0 +1,119 @@
+dist:
+  module: github.com/open-telemetry/opentelemetry-collector-releases/contrib
+  name: otelcol-contrib
+  description: OpenTelemetry Collector Contrib
+  version: 0.41.0
+  output_path: ./_build
+  otelcol_version: 0.41.0
+
+extensions:
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/extension/asapauthextension v0.41.0
```

Comment:
The components were obtained by a mix of sed and awk commands and I verified manually by counting the number of components. An extra pair of eyes won't hurt though.

Reply:
Any thoughts on how to keep this list up to date in the future?

Reply:
Manually reviewed the list and it looks good 👍

Reply:
We are not going to have a components.go in the contrib anymore, so, changing the documentation to point to this instead of that would represent the current status. But the current status isn't good enough: I think we do have components that are ready but not included in the distribution.

Reply:
I agree. It's confusing to have components in the repo, but not in the release without a more clear signal as to why that is... maybe we could start adding all components in this manifest, with the components that are not "ready" in a commented out section? This would at least give users looking for specific components a chance to discover why the component is not there themselves.

Reply:
May I do this in a follow-up PR, if there are no other concerns?

Reply:
works for me.

Reply:
Here's the list of missing components:

```

Extensions
- github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/dockerobserver
- github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/ecsobserver
- github.com/open-telemetry/opentelemetry-collector-contrib/extension/observer/ecstaskobserver

Exporters
- github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awscloudwatchlogsexporter
- github.com/open-telemetry/opentelemetry-collector-contrib/exporter/coralogixexporter
- github.com/open-telemetry/opentelemetry-collector-contrib/exporter/elasticsearchexporter
- github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudpubsubexporter
- github.com/open-telemetry/opentelemetry-collector-contrib/exporter/observiqexporter

Processors
- github.com/open-telemetry/opentelemetry-collector-contrib/processor/redactionprocessor

Receivers
- github.com/open-telemetry/opentelemetry-collector-contrib/receiver/apachereceiver
- github.com/open-telemetry/opentelemetry-collector-contrib/receiver/cloudfoundryreceiver
- github.com/open-telemetry/opentelemetry-collector-contrib/receiver/googlecloudpubsubreceiver
- github.com/open-telemetry/opentelemetry-collector-contrib/receiver/k8seventsreceiver
- github.com/open-telemetry/opentelemetry-collector-contrib/receiver/mysqlreceiver
- github.com/open-telemetry/opentelemetry-collector-contrib/receiver/nginxreceiver
- github.com/open-telemetry/opentelemetry-collector-contrib/receiver/postgresqlreceiver
```